extensions/amp-story/1.0/amp-story.js

```diff
@@ -1814,6 +1815,7 @@ export class AmpStory extends AMP.BaseElement {
           this.element.removeAttribute('desktop');
           this.element.classList.remove('i-amphtml-story-desktop-panels');
           this.element.classList.remove('i-amphtml-story-desktop-fullbleed');
+          this.element.classList.remove('i-amphtml-story-desktop-one-panel');
```

Comment:
Plz also remove this class when switching to another UI type, that could help with your "toggle experiment" button when you're on desktop
The only one we don't really care about is vertical.